docs/user-guide/building-assistants.rst

```diff
@@ -155,37 +155,65 @@ For example:
 
 
 While it's good to test the bot interactively, we should also add end to end test cases that
-can later be included as part of our CI/CD system. `End to end stories <https://rasa.com/docs/rasa/user-guide/testing-your-assistant/#end-to-end-testing>`_
-include NLU data, so that both components of Rasa can be tested.  Create a file called
-``test_stories.md`` in the root directory with some test cases:
+can later be included as part of our CI/CD system. End-to-end :ref:`test conversations <end-to-end-testing>`
+include NLU data, so that both components of Rasa can be tested. The file 
+``tests/conversation_tests.md`` contains test conversations for the stories defined so far:
 
 .. code-block:: md
-
-   ## greet + goodbye
-   * greet: Hi!
-     - utter_greet
-   * bye: Bye
-     - utter_bye
-
-   ## greet + thanks
-   * greet: Hello there
-     - utter_greet
-   * thank: thanks a bunch
-     - utter_noworries
-
-   ## greet + thanks + goodbye
-   * greet: Hey
-     - utter_greet
-   * thank: thank you
-     - utter_noworries
-   * bye: bye bye
-     - utter_bye
+## happy path 1
+* greet: hello there!
+  - utter_greet
+* mood_great: amazing
+  - utter_happy
+
+## happy path 2
+* greet: hello there!
+  - utter_greet
+* mood_great: amazing
+  - utter_happy
+* goodbye: bye-bye!
+  - utter_goodbye
+
+## sad path 1
+* greet: hello
+  - utter_greet
+* mood_unhappy: not good
+  - utter_cheer_up
+  - utter_did_that_help
+* affirm: yes
+  - utter_happy
+
+## sad path 2
+* greet: hello
+  - utter_greet
+* mood_unhappy: not good
+  - utter_cheer_up
+  - utter_did_that_help
+* deny: not really
+  - utter_goodbye
+
+## sad path 3
+* greet: hi
+  - utter_greet
+* mood_unhappy: very terrible
+  - utter_cheer_up
+  - utter_did_that_help
+* deny: no
+  - utter_goodbye
+
+## say goodbye
+* goodbye: bye-bye!
+  - utter_goodbye
+
+## bot challenge
+* bot_challenge: are you a bot?
+  - utter_iamabot
```

Comment:
Hm but these test stories cover too much for the existing data, we should still overwrite them. Sorry, i wasn't clear about that 😅just mean we should say "remove the existing test conversations from tests/conversation_tests.md and add some ones for this convo (same as in L48, we delete all the content -- i guess we could do the deletion there as well)

Reply:
Ok, gotcha now.

Reply:
Does that work?

Reply:
yep!